### PR TITLE
Add examples resources to install

### PR DESCRIPTION
**Pull request by jparisu**
Signed-off-by: jparisu <javierparis@eprosima.com>

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -194,3 +194,12 @@ if (LOG_INFO)
...
 endif()
+
+###############################################################################
+# Install Resources
+###############################################################################
+# Install example files
+
+INSTALL(
+    DIRECTORY ${PROJECT_SOURCE_DIR}/resources/configurations/examples/
+    DESTINATION ${BIN_INSTALL_DIR}/resources/)
```
Comment (juanlofer-eprosima) [inline CMakeLists.txt:205]: Perhaps this location is not the best one, it appears a bit hidden to me.

**Review by juanlofer-eprosima — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (jparisu) [reply CMakeLists.txt]: :+1:

**Review by juanlofer-eprosima — APPROVED** [review]
LGTM, please rebase

**Review by juanlofer-eprosima — APPROVED** [review]
LGTM

**Pull request merged**